src/main/resources/mappings/opendistro-ism-config.json

```diff
@@ -454,6 +454,51 @@
             "last_updated_time": {
               "type": "date",
               "format": "strict_date_time||epoch_millis"
+            },
+            "user": {
+              "properties": {
+                "name": {
+                  "type": "text",
+                  "fields": {
+                    "keyword": {
+                      "type": "keyword",
+                      "ignore_above": 256
+                    }
+                  }
+                },
+                "backend_roles": {
+                  "type": "text",
+                  "fields": {
+                    "keyword": {
+                      "type": "keyword"
+                    }
+                  }
+                },
+                "roles": {
+                  "type": "text",
+                  "fields": {
+                    "keyword": {
+                      "type": "keyword"
+                    }
+                  }
+                },
+                "custom_attribute_names": {
+                  "type": "text",
+                  "fields": {
+                    "keyword": {
+                      "type": "keyword"
+                    }
+                  }
+                },
+                "user_requested_tenant": {
```

Comment:
What's this?

Reply:
actually I can remove this, this part should be part of User object. But since we don't use this, and cannot see any reason use this in future, so in the code, we only keep the other four fields.